### PR TITLE
Autohide bookmarks

### PR DESCRIPTION
**Pull request by andreicristianpetcu**

### CODE REVIEW

**Comment by Timvde** [comment]
I share the concern you outed [in this thread](https://www.reddit.com/r/FirefoxCSS/comments/71sim1/how_can_i_use_a_selector_based_on_url_i_want_to/dnd4kx3/). It is currently English-only and Nightly-only. I'm first going to wait whether the original author or someone else comes with a solution. I would also like to see explicit approval from the author that it is okay for them to merge this (and implicitly convert the style to GPL).

**Comment by andreicristianpetcu** [comment]
Done 
screenshot from 2017-09-23 20-55-14

**Comment by nt1m** [comment]
@Timvde asked me if there was a way to work around the english only restriction in the CSS.

This would actually be possible with CSS Selectors 4, but that spec is still a working draft, so I wouldn't expect it to be implemented it soon.

The best bet here to work around the english only restriction would be creating a webextension that sets titlePreface on the window that has about:newtab open. Then, the about:newtab window would be targetable with CSS.

File: toolbars/show-bookmarks-only-on-newtab.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ * Screenshot: https://vimeo.com/235059188
+ * 
+ * Contributor(s): https://www.reddit.com/user/AJtfM7zT4tJdaZsm and Andrei Cristian Petcu
+ */
+
+#main-window:not([title^="New Tab"]) #PersonalToolbar {
+  visibility: collapse !important;
+}
+
+#main-window:not([title^="Nightly"]) #PersonalToolbar {
```
Comment (nt1m) [inline toolbars/show-bookmarks-only-on-newtab.css]: This rule can be combined with the one above.

**Comment by Timvde** [comment]
So, I've been thinking about this, and I don't think that there's an easy way around this. So I'll merge this as-is, with one extra request: could you change `Nightly` to `Mozilla Firefox`? This will match the behaviour on release and beta, and Nightly users are more likely to be technical and find out what to change.

**Comment by andreicristianpetcu** [comment]
I can add selectors for all.

**Comment by Timvde** [comment]
Oh, lol. Why didn't I even think of that :P Sounds good :)

**Comment by Timvde** [comment]
@andreicristianpetcu If you add selectors for all Firefox versions and process @nt1m's comment, I'll merge right away :)

**Comment by andreicristianpetcu** [comment]
I made [the extension](https://github.com/andreicristianpetcu/autohide_after_newtab/releases/tag/1.0.0)

**Comment by andreicristianpetcu** [comment]
I published the extension in AMO https://addons.mozilla.org/en-US/firefox/addon/autohideafternewtab/

**Comment by andreicristianpetcu** [comment]
any update on this? I did what I was asked :D

**Comment by Timvde** [comment]
I must have missed your last commit, sorry :) I'll test this tomorrow and merge it if it works ;)

**Comment by Timvde** [comment]
Seems to work, nice :)

**Pull request merged**